### ISSUE TITLE
"Fork me on GitHub" on site links to mojohaus repository

### ISSUE THREAD

**Issue by pzygielo** [dev-aspectj/aspectj-maven-plugin#83]
The ribbon _Fork me on GitHub_ on [site](https://dev-aspectj.github.io/aspectj-maven-plugin/) links to [mojohaus repository](https://github.com/mojohaus/aspectj-maven-plugin), instead of [this one](https://github.com/dev-aspectj/aspectj-maven-plugin).

It might be hard to contribute here when following the current link as it will result in forking different repository, not GitHub-linked to this one.

**Comment by kriegaex** [comment]
Yes, your are right. This has historic reasons. For a while, I tried to discuss with the Mojohaus maintainers if we could unify the two forks again after Mojohaus did not maintain the plugin for years and nobody reacted to pull requests or issues. Then suddenly, after I started releasing new versions, finally supporting not just Java 8 but everything up to 16 (now 18), they also released a version for Java 16, merging a few PRs. But some of my improvements were rejected, and finally I decided to continue maintaining this one, because Mojohaus refused to retire their repository. I put off decisions like removing too many Mojohaus links or reformatting code to the way I prefer it, because silently I was still hoping for them to come around, but that never happened. So we still have two forks, theirs always lagging behind this one functionally. But maybe it is about time to cut off the old links, even though I would still prefer the Mojohaus people to retire their version and give users what they want: a single source for the AspectJ Maven plugin. I simply do not want to kick the features they refuse to merge in the trash, though.
